.github/label-actions.yml

```diff
@@ -164,4 +164,19 @@
     Good luck,
 
 
+    The Renovate team
+
+'invalid':
+  comment: >
+    Hi there,
+
+
+    This issue has been marked as invalid and will be closed and potentially deleted too.
+    Deletion is done so that invalid issues (such as confused bug reports) don't show up in repository searches and confuse users.
```

Comment:
```suggestion
    Deletion is done so that invalid issues (such as mistaken bug reports) don't show up in repository searches and confuse users.
```